selenium_tests/lti_emailer/page_objects/add_user_po.py

```diff
@@ -0,0 +1,43 @@
+__author__ = 'hic048'
+
+from selenium.webdriver.common.by import By
+from selenium_tests.lti_emailer.page_objects.emailer_base import EmailerBasePage
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.support.ui import Select
+
+# These are page-specific locators.
+class Locator3(object):
+      _add_people_url = "http://canvas.icommons.harvard.edu/courses/6389/users"
+      _switchframe = (By.ID, "tool_content")
+      _add_user_button = (By.ID, "addUsers")
+      _addtextarea = (By.ID, "user_list_textarea")
+      _next_button = (By.ID, "next-step")
+      _create_user = (By.ID, "createUsersAddButton")
+      _confirm_create = (By.XPATH, "//button[@type='button']")
```

Comment:
I think our convention thus far is to use CSS selectors; would be good to be consistent if possible. We can certainly revisit that convention if you wish.